### DISCUSSION TITLE
Nombre de réunion zoom

### DISCUSSION THREAD

**Discussion by CedricGeorg** [sircharlo/jw-launcher#68] [Ideas]
Bonjour,
L'application est vraiment trés utile, mais mon assemblée utilise plus de 3 numéros de réunion zoom.
Est il possible d'en parametrer 6 ? 

Merci d'avance
Cédric

**Comment by sircharlo** [comment]
Re-bonjour,
Content qu'elle soit utile pour vous! C'est une idée intéressante; je vais voir si je peux faire le changement d'ici la fin du mois.

**Comment by CedricGeorg** [comment]
Mon Papa est malvoyant, mais il est ancien également et c’est vraiment l’application qui lui faut. 
Et personnellement je trouve ça super pratique pour les compagnons en difficulté.

**Reply by sircharlo** [reply]
Merci pour les beaux mots!
